src/pykeen/losses.py

```diff
@@ -1399,6 +1440,41 @@ def forward(
         return loss
 
 
+class AdversarialBCELoss(AdversarialLoss):
```

Comment:
should this be named AdversarialBCEWithLogitsLoss to better mirror the other BCEWithLogitsLoss?